src/main/java/org/openstreetmap/atlas/geography/atlas/items/Node.java

```diff
@@ -29,6 +33,34 @@ public long absoluteValence()
         return this.connectedEdges().size();
     }
 
+    @Override
+    public JsonObject asGeoJsonFeature()
+    {
+        final JsonObject feature = super.asGeoJsonFeature();
+        final JsonObject properties = feature.getAsJsonObject("properties");
```

Comment:
same ad before. This function could just be overriding the ```getGeojsonProperties()``` function.

Reply:
Addressed ^^^.